<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GeothermMainWindow</class>
 <widget class="QMainWindow" name="GeothermMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>949</width>
    <height>985</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GeothermMainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QTabWidget" name="views">
        <property name="currentIndex">
         <number>1</number>
        </property>
        <widget class="QWidget" name="input">
         <attribute name="title">
          <string>Input</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QLabel" name="label_8">
              <property name="text">
               <string>Data file, CSV:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="dataFileName">
              <property name="text">
               <string>/home/eugeneai/projects/code/dymsh/data/PTdata.csv</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton">
              <property name="text">
               <string>Choose...</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton_2">
              <property name="text">
               <string>Load</string>
              </property>
              <property name="default">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QGroupBox" name="groupBox">
            <property name="title">
             <string>Input Data</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_3">
             <item>
              <widget class="QTableView" name="csvView"/>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="init">
         <attribute name="title">
          <string>Initials</string>
         </attribute>
         <layout class="QGridLayout" name="gridLayout">
          <item row="7" column="3">
           <widget class="QLineEdit" name="H">
            <property name="text">
             <string>[0,0.4,0.4,0.02]</string>
            </property>
           </widget>
          </item>
          <item row="6" column="3">
           <widget class="QLineEdit" name="P">
            <property name="text">
             <string>0.74</string>
            </property>
           </widget>
          </item>
          <item row="1" column="3">
           <widget class="QLineEdit" name="q0">
            <property name="text">
             <string>34:1:40</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0" colspan="2">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>q0</string>
            </property>
            <property name="buddy">
             <cstring>q0</cstring>
            </property>
           </widget>
          </item>
          <item row="7" column="0">
           <widget class="QLabel" name="label_7">
            <property name="text">
             <string>H</string>
            </property>
            <property name="buddy">
             <cstring>H</cstring>
            </property>
           </widget>
          </item>
          <item row="4" column="3">
           <widget class="QLineEdit" name="Zmax">
            <property name="text">
             <string>255</string>
            </property>
           </widget>
          </item>
          <item row="5" column="3">
           <widget class="QLineEdit" name="Dz">
            <property name="text">
             <string>0.1</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Zbot</string>
            </property>
            <property name="buddy">
             <cstring>Zbot</cstring>
            </property>
           </widget>
          </item>
          <item row="5" column="0">
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>Dz</string>
            </property>
            <property name="buddy">
             <cstring>Dz</cstring>
            </property>
           </widget>
          </item>
          <item row="8" column="3">
           <widget class="QLineEdit" name="iref">
            <property name="text">
             <string>3</string>
            </property>
           </widget>
          </item>
          <item row="8" column="0">
           <widget class="QLabel" name="label_15">
            <property name="text">
             <string>iref</string>
            </property>
            <property name="buddy">
             <cstring>iref</cstring>
            </property>
           </widget>
          </item>
          <item row="2" column="3">
           <widget class="QLineEdit" name="D">
            <property name="text">
             <string>16</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>D</string>
            </property>
            <property name="buddy">
             <cstring>D</cstring>
            </property>
           </widget>
          </item>
          <item row="6" column="0">
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>P</string>
            </property>
            <property name="buddy">
             <cstring>P</cstring>
            </property>
           </widget>
          </item>
          <item row="3" column="3">
           <widget class="QLineEdit" name="Zbot">
            <property name="text">
             <string>[16,23,39,300]</string>
            </property>
           </widget>
          </item>
          <item row="11" column="3">
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>Zmax</string>
            </property>
            <property name="buddy">
             <cstring>Zmax</cstring>
            </property>
           </widget>
          </item>
          <item row="0" column="3">
           <widget class="QLabel" name="label_9">
            <property name="text">
             <string>Values of the fields are evaluated as Julia expressions within loaded sources, no check made.</string>
            </property>
           </widget>
          </item>
          <item row="9" column="6">
           <widget class="QPushButton" name="calcPushButton">
            <property name="text">
             <string>Calculate</string>
            </property>
           </widget>
          </item>
          <item row="4" column="6">
           <widget class="QPushButton" name="pushButton_3">
            <property name="text">
             <string>Check Data</string>
            </property>
           </widget>
          </item>
          <item row="1" column="6">
           <widget class="QCheckBox" name="opt">
            <property name="text">
             <string>Optimize</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="tableOutput">
         <attribute name="title">
          <string>Output, table</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <item>
           <widget class="QGroupBox" name="groupBox_2">
            <property name="title">
             <string>Geotherm calculation results, tabular view</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_4">
             <item>
              <widget class="QTableView" name="csvResultView"/>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="graphicsOutput">
         <attribute name="title">
          <string>Output, graphics</string>
         </attribute>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>949</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="separator"/>
    <addaction name="action_Load"/>
    <addaction name="action_Close"/>
    <addaction name="separator"/>
    <addaction name="action_Quit"/>
   </widget>
   <widget class="QMenu" name="menu_View">
    <property name="title">
     <string>&amp;View</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu_Report">
    <property name="title">
     <string>&amp;Report</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu_About">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="actionProject_Home_page"/>
    <addaction name="separator"/>
    <addaction name="action_About"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menu_View"/>
   <addaction name="menu_Report"/>
   <addaction name="menu_About"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="actionProject_Home_page">
   <property name="text">
    <string>Project &amp;Home page</string>
   </property>
  </action>
  <action name="action_About">
   <property name="text">
    <string>&amp;About</string>
   </property>
  </action>
  <action name="action_Load">
   <property name="text">
    <string>&amp;Load</string>
   </property>
  </action>
  <action name="action_Quit">
   <property name="icon">
    <iconset theme="application-exit">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>&amp;Quit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="action_Close">
   <property name="text">
    <string>&amp;Close</string>
   </property>
  </action>
 </widget>
 <tabstops>
  <tabstop>q0</tabstop>
  <tabstop>D</tabstop>
  <tabstop>Zbot</tabstop>
  <tabstop>Zmax</tabstop>
  <tabstop>Dz</tabstop>
  <tabstop>P</tabstop>
  <tabstop>H</tabstop>
  <tabstop>iref</tabstop>
  <tabstop>opt</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>action_Quit</sender>
   <signal>triggered()</signal>
   <receiver>GeothermMainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>action_About</sender>
   <signal>triggered()</signal>
   <receiver>GeothermMainWindow</receiver>
   <slot>actionAbout()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>actionAbout()</slot>
 </slots>
</ui>
